src/main/java/de/zalando/tip/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -271,11 +278,12 @@ void copyVideos(final String containerId) throws IOException, DockerException, I
         LOGGER.log(Level.INFO, "{0} Video files copies to: {1}", new Object[]{getNodeIpAndPort(), localPath});
     }
 
-    DockerSeleniumNodePoller getDockerSeleniumNodePollerThread() {
+    @VisibleForTesting
+    protected DockerSeleniumNodePoller getDockerSeleniumNodePollerThread() {
         return dockerSeleniumNodePollerThread;
     }
 
-    boolean isStopSessionRequestReceived() {
+    private boolean isStopSessionRequestReceived() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this method into "DockerSeleniumNodePoller". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3398)